requirements.txt

```diff
@@ -0,0 +1,4 @@
+# NOTE: This project does not use requirements.txt, it uses setup.py.
```

Comment:
I suspect some tooling will use this and it will cause issues. Why can't we just install it manually for Travis?